plugins/s3-viewer-backend/src/service/plugin.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  BucketStatsProvider,
+  BucketsProvider,
+  CredentialsProvider,
+  S3Api,
+  s3ViewerExtensionPoint,
+} from '@spreadshirt/backstage-plugin-s3-viewer-node';
+import {
+  coreServices,
+  createBackendPlugin,
+} from '@backstage/backend-plugin-api';
+import { S3Builder } from './S3Builder';
+import express from 'express';
+
+export const s3ViewerPlugin = createBackendPlugin({
+  pluginId: 's3',
```

Comment:
Let's call this `s3-viewer`, to leave some space for other non-viewer plugins related to S3.

Reply:
I'll do that. However, I think we need to make sure the frontend can connect to the plugin. Because now it search for the plugin `s3`, but it will be renamed to `s3-viewer`. Will change both things

Reply:
This has been fixed in the last commit